package.json

```diff
@@ -44,11 +44,11 @@
     "start": "node index.js",
     "sonar-scan": "node_modules/sonar-scanner/bin/sonar-scanner",
     "generate-json": "yarn --cwd ccd-definition-processor xlsx2json -D ../definitions/divorce/json -i ../definitions/divorce/xlsx/ccd-config-base.xlsx && pretty-quick --pattern 'definitions/**/json/**.json'",
-    "generate-excel-local": "DIV_ENV=local CCD_DEF_COS_URL=$npm_package_config_local_cosUrl CCD_DEF_CCD_URL=$npm_package_config_local_ccdUrl yarn run generate-excel",
-    "generate-excel-demo": "DIV_ENV=demo CCD_DEF_COS_URL=$npm_package_config_demo_cosUrl CCD_DEF_CCD_URL=$npm_package_config_demo_ccdUrl yarn run generate-excel",
-    "generate-excel-aat": "DIV_ENV=aat CCD_DEF_COS_URL=$npm_package_config_aat_cosUrl CCD_DEF_CCD_URL=$npm_package_config_aat_ccdUrl yarn run generate-excel",
-    "generate-excel-ithc": "DIV_ENV=ithc CCD_DEF_COS_URL=$npm_package_config_ithc_cosUrl CCD_DEF_CCD_URL=$npm_package_config_ithc_ccdUrl yarn run generate-excel",
-    "generate-excel-prod": "DIV_ENV=prod CCD_DEF_COS_URL=$npm_package_config_prod_cosUrl CCD_DEF_CCD_URL=$npm_package_config_prod_ccdUrl yarn run generate-excel -e *-nonprod.json",
+    "generate-excel-local": "DIV_ENV=local CCD_DEF_COS_URL=$npm_package_config_local_cosUrl CCD_DEF_CCD_URL=$npm_package_config_local_ccdUrl yarn run generate-excel -e *-only-for-prod.json",
+    "generate-excel-demo": "DIV_ENV=demo CCD_DEF_COS_URL=$npm_package_config_demo_cosUrl CCD_DEF_CCD_URL=$npm_package_config_demo_ccdUrl yarn run generate-excel -e *-only-for-prod.json",
+    "generate-excel-aat": "DIV_ENV=aat CCD_DEF_COS_URL=$npm_package_config_aat_cosUrl CCD_DEF_CCD_URL=$npm_package_config_aat_ccdUrl yarn run generate-excel -e *-only-for-prod.json",
+    "generate-excel-ithc": "DIV_ENV=ithc CCD_DEF_COS_URL=$npm_package_config_ithc_cosUrl CCD_DEF_CCD_URL=$npm_package_config_ithc_ccdUrl yarn run generate-excel -e *-only-for-prod.json",
+    "generate-excel-prod": "DIV_ENV=prod CCD_DEF_COS_URL=$npm_package_config_prod_cosUrl CCD_DEF_CCD_URL=$npm_package_config_prod_ccdUrl yarn run generate-excel -e *-only-for-nonprod.json",
```

Comment:
Should be:
```
    "generate-excel-prod": "DIV_ENV=prod CCD_DEF_COS_URL=$npm_package_config_prod_cosUrl CCD_DEF_CCD_URL=$npm_package_config_prod_ccdUrl yarn run generate-excel -e *-nonprod.json",
```

Reply:
I changed the name of this to be more consistent with the other variable. Is there a benefit for the names not to align?

Reply:
I'd leave "nonprod" as it has become a standard accross several repos and projects (FR, FPLA, us).

Also we already had a presentation about "nonprod" approach for all devs in div and PET team and I'm sure it will cause additional problems in the future if we change it. 

Too much noise, little gain. But you're right - this would be better if we (or FLPA in the first place) have done it like this at the beginning.

Reply:
Im a bit confused and I'm unsure what the issue with this is. 

The tags are `only-for-prod` and `only-for-nonprod`. Both are clear that each feature is on its respected environments. I haven't removed the terminology of nonprod, but specified that whatever is included in a specific file is exclusive to it

Reply:
Have other teams already implemented something like this? And if so, but used poor naming conventions, should we not encourage them to refactor this as it is best practice

Reply:
ok, so change it:
"-prod.json"
and
"-nonprod.json".

Reply:
Updated